js/controllers/linkAction/LinkActionHideButtons.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @defgroup js_controllers_linkAction
+ */
+/**
+ * @file js/controllers/linkAction/LinkActionHideButtons.js
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class EditorDecisionHideButtons
+ * @ingroup js_controllers_linkAction_LinkActionHideButtons
+ *
+ * @brief Hide decision buttons (depending on stage) after a decision has been recorded.
+ */
+(function($) {
+
+	/** @type {Object} */
+	$.pkp.controllers = $.pkp.controllers ||
+			{ linkAction: {LinkActionHideButtons: { } } };
+
+	/**
+	 * @constructor
+	 *
+	 * @extends $.pkp.controllers
+	 */
+	$.pkp.controllers.LinkAction_LinkActionHideButtons = function() {
+
+		alert('include script code here, somehow?')
+
+	};
+
+/** @param {jQuery} $ jQuery closure. */
+}(jQuery));
```

Comment:
**Note** this is just how I think the java script part might be included? You told me that you'll help me with that if needed, seems it is needed *g*. I think I figured out how how it basically works, but could not figure out how to get it running. I think I have to compile the js library (npm?). Where should this script be located? I just put it into "controllers/linkAction/LinkActionHideButtons" (t.b.d.).